plugins/dstat_nvidia_gpu.py

```diff
@@ -0,0 +1,41 @@
+### Author: Vasilis Vryniotis <bbriniotis@datumbox.com>
+
+class dstat_plugin(dstat):
+    """
+    Total GPU usage for NVIDIA cards. Requires the nvidia-ml-py package.
+
+    Usage: dstat --nvidia-gpu
+    """
+
+    def __init__(self):
+        self.name = 'gpu nvidia'
+        self.nick = ('gpu',)
+        self.vars = ('gpu',)
+        self.type = 'p'
+        self.width = 5
+        self.scale = 1
+
+    def check(self):
+        import pynvml
+        pynvml.nvmlInit()
+
+    def extract(self):
+        self.val['gpu'] = self._getTotalUsage(10)
+
+    def _getUsagePerGPU(self, samples):
```

Comment:
I don't know if it is useful (and I let you decide), but if you have per-cpu statistics, maybe some people would like to see them for each of their GPUs.